config/locales/en.yml

```diff
@@ -257,6 +257,9 @@ en:
               reused:
                 one: "has been used before. Please choose one that is different from your last one."
                 other: "has been used before. Please choose one that is different from your last %{count}."
+              match: >
+                Confirm new password.
+                Password should match with the one you have typed in New password field.
```

Comment:
Still not a fan of having line breaks done in the yaml-localisation. Better way would probably be something like breaking this up into two labels, as `match` implies only the match-validator. `Confirm new password` reads more like a separate label to me.